doc/classes/LineEdit.xml

```diff
@@ -133,6 +133,18 @@
 				[b]Warning:[/b] This is a required internal node, removing and freeing it may cause a crash. If you wish to hide it or any of its children, use their [member Window.visible] property.
 			</description>
 		</method>
+		<method name="get_next_composite_character_column" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the correct column at the end of a composite character like ❤️‍🩹 (mending heart; Unicode: U+2764 U+FE0F U+200D U+1FA79) which is comprised of more than one unicode, if the caret is at the start of the composite character.
```

Comment:
```suggestion
				Returns the correct column at the end of a composite character like ❤️‍🩹 (mending heart; Unicode: [code]U+2764 U+FE0F U+200D U+1FA79[/code]) which is comprised of more than one unicode, if the caret is at the start of the composite character.
```
Also is "one unicode" really established terminology?

Reply:
Probably should be "more than one Unicode [code point](https://unicode.org/glossary/#code_point)".